h5c/vic/src/vic-webapp/src/app/create-vch-wizard/compute-capacity/compute-capacity.component.spec.ts

```diff
@@ -147,7 +147,7 @@ describe('ComputeCapacityComponent', () => {
     // Validate result
     setDefaultRequiredValues();
     component.onCommit().subscribe( r => {
-      expect(r.computeCapacity.cpu).toBe('1');
```

Comment:
Won't this changes break the CLI command which is autogenerated based on this property names?